arxiv/base/templates/base/base.html

```diff
@@ -8,7 +8,7 @@
   </head>
   <body>
   {% import "base/macros.html" as macros %}
-  <noscript><img src="//webanalytics.library.cornell.edu/piwik.php?idsite=538&rec=1" style="border:0;" alt="" /></noscript>
+  <noscript><p><img src="https://webstats.arxiv.org/matomo.php?idsite=1&amp;rec=1" style="border:0;" alt="" /></p></noscript>
```

Comment:
Any chance we can make this (and maybe the domains/URLs further down) configurable? Would save having to mint new software versions to change this in the future. Not that it should change often, but still.

Reply:
I'd argue that for this particular case alone, setting the domain for Matomo in a config variable may not be the best choice - it is highly unlikely to change and may have as many subtle cons as pros. Would obfuscate the URL for future developers and make the tracker harder to find, increasing possibility for something to go wrong in config and take down tracking without meaning to, ending up with a data loss.

It's a minor point, and if there's a strong reason to use config that would be fine too.